secrets.tf

```diff
@@ -1,13 +1,15 @@
 locals {
   grafana_admin_user_key     = "admin-user"
   grafana_admin_password_key = "admin-password" #pragma: allowlist secret
+  grafana_namespace          = coalesce(var.namespace, try(local.addon.namespace, null), local.addon.name)
+  grafana_secret_name        = coalesce(try(local.addon.helm_chart_name, null), local.addon.name)
 }
 
 resource "kubernetes_namespace" "grafana" {
```

Comment:
```suggestion
resource "kubernetes_namespace_v1" "grafana" {
```

Reply:
I am not sure here, but would not this cause re-creation of the Namespace? If so, can we avoid that somehow?